backend/gateway/src/locales/en/frontline.json

```
@@ -815,6 +815,13 @@
   "assigned-user": "Assigned User",
   "priority-label": "Priority",
   "properties-label": "Properties",
+  "group-by-label": "Group by",
+  "no-grouping": "No grouping",
+  "all": "All",
+  "total-tickets": "Total tickets",
```

Biome (2.5.5)

[error] The key total-tickets was already declared.

(lint/suspicious/noDuplicateObjectKeys)